src/ophyd_async/epics/demo/__init__.py

```diff
@@ -43,6 +49,19 @@ def __init__(self, prefix: str, name="") -> None:
         super().__init__(name=name)
 
 
+class SensorGroup(StandardReadable):
```

Comment:
Does the ioc subprocess command at the bottom of the file actually contain all these sensors? I'd like to keep this file and the tutorials in step so that you can always run up an IOC to test the example code...

Reply:
It does now, although I think we should document that process/make it easier. Maybe even add tests to prove that the IOC contains the demo objects. I'll make an issue or two.

Reply:
Done: https://github.com/bluesky/ophyd-async/issues/193